tests/specs/getLocalChanges.js

```diff
@@ -89,8 +84,62 @@ test('getLocalChanges(string)', function (t) {
   .then(function () {
     return api.getLocalChanges('test2')
   })
-  .then(function (arr) {
-    console.log(arr[0])
-    t.equal(arr[0] > arr[1], true, 'local changes detected')
+  .then(function (localReplicationDoc) {
+    t.equal(localReplicationDoc.length, 1, '1 local changes detected with _id:"test2"')
+  })
+})
+
+test('getLocalChanges(obj)', function (t) {
+  t.plan(1)
+  var db1 = dbFactory('LCDB7')
+  var api = db1.hoodieLocalChanges({local: 'LCDB7', remote: 'LCDB8'})
+
+  var localObj1 = {_id: 'test1', foo1: 'bar1'}
+  var localObj2 = {_id: 'test2', foo1: 'bar2'}
+  var localObj3 = {_id: 'test3', foo1: 'bar3'}
+
+  db1.bulkDocs([localObj1, localObj2, localObj3])
+  .then(function () {
+    return api.getLocalChanges(localObj1)
+  })
+  .then(function (localReplicationDoc) {
+    t.equal(localReplicationDoc.length, 1, '1 local changes detected with _id:"test1"')
+  })
+})
+
+test('getLocalChanges(objs)', function (t) {
+  t.plan(1)
+  var db1 = dbFactory('LCDB9')
+  var api = db1.hoodieLocalChanges({local: 'LCDB9', remote: 'LCDB10'})
+
+  var localObj1 = {_id: 'test1', foo1: 'bar1'}
+  var localObj2 = {_id: 'test2', foo1: 'bar2'}
+  var localObj3 = {_id: 'test3', foo1: 'bar3'}
+
+  db1.bulkDocs([localObj1, localObj2, localObj3])
+  .then(function () {
+    return api.getLocalChanges([localObj3, 'test2'])
+  })
+  .then(function (localReplicationDoc) {
+    t.equal(localReplicationDoc.length, 2, '2 local changes detected with obj 3 and _id:"test2"')
+  })
+})
+
+test('getLocalChanges(multiple objs)', function (t) {
```

Comment:
yeah, `test('getLocalChanges(multiple objs)'` is the same as `[localObj3, localObj2]`, right? We don't need to test with an  array of two objects, if you already test with an array of an object and an id